### PR TITLE
feat: new repo for wallet-dim-stub application

### PR DESCRIPTION
**Pull request by nitin-vavdiya**
## Description
New repository setup for wallet DIM stub application 

ref: https://github.com/eclipse-tractusx/sig-release/issues/788

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: otterdog/eclipse-tractusx.jsonnet [hunk 1]
```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
+      description: "Wallet DIM stub application",
+      allow_forking: true,
```
Comment (evegufy) [inline otterdog/eclipse-tractusx.jsonnet:1522]: should not be necessary, as the value defaults to `true`

File: otterdog/eclipse-tractusx.jsonnet [hunk 2]
```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
...
+      allow_merge_commit: true,
+      dependabot_alerts_enabled: true,
+      dependabot_security_updates_enabled: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      default_branch: "main",
```
Comment (evegufy) [inline otterdog/eclipse-tractusx.jsonnet:1528]: should not be necessary, as the value defaults to main

File: otterdog/eclipse-tractusx.jsonnet [hunk 3]
```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
...
+      has_discussions: true,
+      has_issues: true,
```
Comment (evegufy) [inline otterdog/eclipse-tractusx.jsonnet:1530]: should not be necessary, as the value defaults to `true`

File: otterdog/eclipse-tractusx.jsonnet [hunk 4]
```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
+      description: "Wallet DIM stub application",
```
Comment (evegufy) [inline otterdog/eclipse-tractusx.jsonnet:1521]: Maybe "Stub application for DIM Wallet" tells a bit more

File: otterdog/eclipse-tractusx.jsonnet [hunk 5]
```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
```
Comment (evegufy) [inline otterdog/eclipse-tractusx.jsonnet:1520]: what do you think about `ssi-dim-wallet-stub`? as it fits in the area of Self-sovereign identities (SSI).
https://github.com/eclipse-tractusx?q=ssi&type=all&language=&sort=

File: otterdog/eclipse-tractusx.jsonnet (hunk 2 above)
Comment (nitin-vavdiya) [reply otterdog/eclipse-tractusx.jsonnet]: Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27

File: otterdog/eclipse-tractusx.jsonnet (hunk 3 above)
Comment (nitin-vavdiya) [reply otterdog/eclipse-tractusx.jsonnet]: Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27

File: otterdog/eclipse-tractusx.jsonnet (hunk 4 above)
Comment (nitin-vavdiya) [reply otterdog/eclipse-tractusx.jsonnet]: Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27

File: otterdog/eclipse-tractusx.jsonnet (hunk 5 above)
Comment (nitin-vavdiya) [reply otterdog/eclipse-tractusx.jsonnet]: Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27

File: otterdog/eclipse-tractusx.jsonnet [hunk 6]
```diff
@@ -1517,5 +1517,19 @@ orgs.newOrg('eclipse-tractusx') {
...
+      allow_merge_commit: true,
+      dependabot_alerts_enabled: true,
+      dependabot_security_updates_enabled: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      has_discussions: true,
+      private_vulnerability_reporting_enabled: true,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```
Comment (netomi) [inline otterdog/eclipse-tractusx.jsonnet:1531]: please use "read" default permissions for new repos.

File: otterdog/eclipse-tractusx.jsonnet (hunk 1 above)
Comment (nitin-vavdiya) [reply otterdog/eclipse-tractusx.jsonnet]: Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/commit/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27

File: otterdog/eclipse-tractusx.jsonnet (hunk 6 above)
Comment (nitin-vavdiya) [reply otterdog/eclipse-tractusx.jsonnet]: Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/4c77d7ab506deac405bb300e0a3dbbef4a92574c

**Review by netomi — APPROVED** [review]

**Review by evegufy — APPROVED** [review]

**Pull request merged**